RELEASE.md

```diff
@@ -1,60 +1,63 @@
 # Release Process
 
-Releases are published to NPM when `develop` is merged into `main` AND when the merged code contains a version bump in the `package.json`.
+Releases are published to npm using [Release Please](https://github.com/googleapis/release-please).
 
-## Create and publish a new release:
+This package automates CHANGELOG generation, version bumps and npm releases by parsing the
+git history, looking for [Conventional Commit messages](https://www.conventionalcommits.org/).
 
-### Bump version
+When changes and feature PRs are merged from develop to main, `release-please` will open and maintain a release PR with the updated CHANGELOG and new version number. When this PR is merged, a release will be created and the package published to NPM.
 
-You can manually trigger the [Bump version](https://github.com/ERC725Alliance/erc725.js/actions/workflows/bump-version.yml) workflow from the `develop` branch.
+1. Merge develop into main.
+2. Release Please will create the release PR going to main.
+3. Merge the generated release PR.
```

Comment:
I think here we need to force the merge because the PR is open by a robot
Maybe worth mentioning?

Reply:
Good one, it is very likely indeed that there is smtgh with CI not running because it was opened by a robot. Maybe we need to adjust the CI so it doesnt have to run for the release PRs so it doesnt block it. Lets give it a try as it is now and we can later improve it.